games/devtest/mods/testformspec/formspec.lua

```diff
@@ -64,6 +64,35 @@ local inv_style_fs = [[
 	list[current_player;main;.5,7;8,4]
 ]]
 
+-- Some textures from textures/base/pack and Devtest, with many different sizes
+-- and aspect ratios.
+local image_column = "image,0=logo.png,1=rare_controls.png,2=checkbox_16.png," ..
+		"3=checkbox_32.png,4=checkbox_64.png,5=default_lava.png," ..
+		"6=progress_bar.png,7=progress_bar_bg.png"
+local words = {
+	"esciunt", "repudiandae", "repellat", "voluptatem", "autem", "vitae", "et",
+	"minima", "quasi", "facere", "nihil", "ea", "nemo", "rem", "non", "eos",
+	"laudantium", "eveniet", "veritatis",
+}
+local table_content = {}
+for i = 1, 100 do
+	table.insert(table_content, words[math.random(#words)])
```

Comment:
Randomness makes it very difficult to compare across versions.
Possible solutions: constant seed or PRNG

I tested it as follows: (re-seeding should not be much of an issue for devtest)
```lua
math.randomseed(1337)
for i = 1, 100 do
	-- ...
end
math.randomseed(os.time())
```